### DISCUSSION TITLE
Aiuto progettazione illuminazione

### DISCUSSION THREAD

**Discussion by lucadallavalle** [indomus/forum#3048] [GENERALE - Discussioni generiche]
Prima di tutto, vi conosco da poco, bella community e anche mi sembra gestita bene. Complimenti!

Ho necessità di progettare le luci della zona giorno di casa nuova, i componenti li devo ancora comprare.
Il mio elettricista non mi è d'aiuto e i requisiti che ho, secondo me, sono piuttosto basici, ma non riesco a trovare info chiare a riguardo.
Ecco i miei requisiti (scusate l'ignoranza, sono piuttosto inesperto ancora):

- Ho 10 faretti, 5 di questi vanno controllati (accesi/spenti) indipendentemtnte dagli altri 5. Quindi 2 interruttori e cavi indipendenti.
- Uso Home Assistant e protocollo Zigbee con ZHA.
- Voglio che siano controllabili sia da remoto che con interruttore. Cioè se per qualche motivo la mia istanza di HA è giù, le luci devono continuare a funzionare con gli interruttori. Però se invece HA funziona e le luci sono spente, devo poter accendere le luci da remoto.

Qual'è la soluzione più pratica? Qualche idea?

- Interruttore Zigbee + luci non smart?
- Interruttore non smart + luci Zigbee gestite come 2 flotte da 5?
- Sia interruttore che luci Zigbee?

Grazie mille.

**Comment by esposimo** [comment]
Ciao, 

cerco di esprimere il mio pensiero aldilà della tecnologia scelta (Zigbee o Wifi)
A mio avviso, tutto dipende dalle luci, faccio alcuni esempi

Deviatore Smart + Luci normali
Il deviatore smart si occupa di aprire e chiudere il circuito per accendere/spegnere la luce. Se è un buon deviatore, sarà anche capace di non fare casini tra lo stato fisico del deviatore e i comandi smart.
Esempio: Luce spenta, deviatore su "off". Premi il deviatore, va su "on", la luce si accende (fin qui come un normale deviatore). Comando remoto (app, HA, quello che sia) , per spegnere, la luce si deve spegnere anche se il deviatore è rimasto su "on". Premo di nuovo il deviatore, che va su "off", la luce deve accendersi.
In sostanza, il deviatore smart con luce non smart deve tenere conto del fatto che gli input arrivano da due "canali" diversi.
Se non hai luci dimmerabili, o che variano di colore, va benissimo.
Se invece il deviatore/interruttore è in aggiunta ai normali deviatori di casa (es. un deviatore smart da "incollare" a parete, a batteria, e che non gestisce l'alimentazione della luce smart in maniera diretta) il problema è simile, il deviatore fisico non deve essere mai usato, stare sempre su on, ed in più la luce deve essere smart

Interruttore non smart + Luci zigbee.
Questa è a mio avviso , e per come intendo io la smart home, la peggiore delle scelte. Un interruttore/deviatore non smart apre e chiude la fase verso il carico (la luce).
Una luce smart, wifi/zigbee che sia, anche se non emana luce, ha bisogno di quel minimo di corrente (parliamo di microW) per attendere il comando di accensione. Un deviatore normale, che spegne la luce (e quindi apre il circuito) , non alimenta più la luce smart, che non potrà essere accesa con comandi vocali, ma solo da deviatore. In sostanza, devi lasciare il deviatore su "on" e non premerlo mai più.

Interruttore+Luci smart
A mio avviso la migliore scelta, soprattutto nel caso in cui il deviatore sia configurabile a tal punto dal non fare "niente" quando lo premi. Nel senso che potrebbe inviare un semplice messaggio su MQTT per dire "mi hanno premuto" e a quel punto far partire una automazione che inverte le luci di quel deviatore (un semplice light.toggle)
Inoltre ti consente, nel futuro, di cambiare le luci smart con altre luci smart (dimmerabili, colorate, etc)
Ritengo questa scelta la migliore perchè ti offre la possibilità di usare le luci sia in modalità "smart-home" che "normale" in modalità mista. E' utile perchè non sei schiavo di una delle due "modalità", e poi  torna utile per quando vengono i genitori a casa o ospiti, e non sono avvezzi alle nuove "tecnologie" e possono usare la casa come han sempre fatto

Prova a valutare anche la seguente soluzione
Deviatore normale comandato da un rele smart (wifi/zigbee) e luce normale.
Se non hai esigenze particolari sulle luci (non sono dimmerabili, non hanno milioni di colori) è un'ottima scelta

**Reply by lucadallavalle** [reply]
Grazie mille del commento.

Provo ad approfondire la questione del relé smart, che sembra una soluzione interessante.

Se ho più interruttori che controllano un punto luce mi basta un solo relé giusto? O me ne serve uno per interruttore?

**Reply by esposimo** [reply]
Si, sempre e solo uno.
Se consideri uno Shelly 1 PM, con un solo canale, puoi avere anche un circuito di 10 interruttori (tra deviatori e invertitori) che accendono un solo punto luce, continueranno a funzionare.

Ti consiglio di guardare questo schema, che è basato su 2 punti luce che hanno 2 deviatori ciascuno (con uno shelly 2 pm)
![image](https://github.com/indomus/forum/assets/13506630/d3487b48-0997-4f20-862e-431fdc0f43f5)

**Reply by lucadallavalle** [reply]
Grazie mille veramente! 🍻

Farò delle prove e ti farò sapere.
